matplotcheck/tests/test_lines.py

```diff
@@ -0,0 +1,178 @@
+"""Tests for the vector module"""
+import pytest
+import matplotlib.pyplot as plt
+import geopandas as gpd
+from shapely.geometry import LineString
+from matplotcheck.vector import VectorTester
+import matplotlib
+
+matplotlib.use("Agg")
+
+
+@pytest.fixture
+def multi_line_gdf(two_line_gdf):
+    """ Create a multi-line GeoDataFrame.
+    This has one multi line and another regular line.
+    """
+    # Create a single and multi line object
+    multiline_feat = two_line_gdf.unary_union
+    linec = LineString([(2, 1), (3, 1), (4, 1), (5, 2)])
+    out_df = gpd.GeoDataFrame(
+        geometry=gpd.GeoSeries([multiline_feat, linec]), crs="epsg:4326",
+    )
+    out_df["attr"] = ["road", "stream"]
+    return out_df
+
+
+@pytest.fixture
+def mixed_type_geo_plot(pd_gdf, multi_line_gdf):
+    """Create a point plot for testing"""
+    _, ax = plt.subplots()
+
+    pd_gdf.plot(ax=ax)
+    multi_line_gdf.plot(ax=ax)
+
+    axis = plt.gca()
```

Comment:
here as well - avoid plt.gca and call ax instead to create your tester object. please check for this throughout